File: C++/1153.cpp
#include <iostream>
using namespace std;

int main(){
    int r = 1, num = 0, i = 1;
    cin >> num;
    for(i = 1; i <= num; i++){
        r = i * r;    
    }
    cout << r << endl;
    return 0;
}

File: java/1153.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int i = 1, r = 1, num = 0;
        num = in.nextInt();
        for(i = 1; i <= num; i++){
            r = r * i;
        }
        System.out.println(r);
    }
}